src/main/java/com/synopsys/integration/alert/common/ConfigurationFieldModelConverter.java

```diff
@@ -116,4 +139,13 @@ public final FieldAccessor convertToFieldAccessor(final FieldModel fieldModel) t
         }
         return Optional.of(configurationModel);
     }
+
+    private boolean areValuesEmptyOrBlank(final Collection<String> values) {
+        if (null == values || values.isEmpty()) {
+            return true;
+        }
+        final Optional<String> first = values.stream().filter(StringUtils::isNotBlank).findFirst();
```

Comment:
You might want to try `return values.stream().allMatch(StringUtils::isBlank);` (or `.anyMatch(...)` depending on what you're testing).